static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -28,13 +28,13 @@ export const getVariant: ABTest => string = test => {
 
 // check if the user has dismissed the gate by checking the user preferences
 export const hasUserDismissedGate: ({
-    componentId?: string,
-    componentName: string,
+    name: string,
```

Comment:
use `?` to indicate optional param?

Reply:
it's not optional anymore, hence I'm not adding the `?`

Reply:
ah yes, it should have a `?` since it can be optional